app/database/db_connection.js

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require('pg')
+const url = require('url')
+require('env2')('./config.env')
+
```

Comment:
could we put a line in here to get a different db_url depending on whether we are in a production or a dev environment?